bot/converters.py

```diff
@@ -215,3 +216,35 @@ class Duration(Converter):
         now = datetime.utcnow()
 
         return now + delta
+
+
+class ISODateTime(Converter):
+    """Converts an ISO-8601 datetime string into a datetime.datetime."""
+
+    async def convert(self, ctx: Context, datetime_string: str) -> datetime:
+        """
+        Converts a ISO-8601 `datetime_string` into a `datetime.datetime` object.
+
+        The converter is flexible in the formats it accepts, as it uses the `isoparse` method of
+        `dateutil.parser`. In general, it accepts datetime strings that start with a date,
+        optionally followed by a time.
+
+        See: <https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.isoparse>
```

Comment:
Why is this between angle brackets?

Reply:
Annoying habit from typing URLs in Discord. I'll change it.